### PR TITLE
Fix #8490 Wrong message in the required message on Policy page

### LINKED ISSUE #8490: Wrong message in the required message on Policy page
**Affected module**
Does it impact the UI, backend or Ingestion Framework?
**To Reproduce**
- edit policy
- remove all operations

https://localhost:8585/settings/access/policies/DataConsumerPolicy/edit-rule/DataConsumerPolicy-EditRule

Screenshots or steps to reproduce
image
image

**Version:**
 - Python version:
 - OpenMetadata version: [e.g. 0.8]
 - OpenMetadata Ingestion package version: [e.g. `openmetadata-ingestion[docker]==XYZ`]

**Additional context**

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
Closes #8490
### Describe your changes :

I worked on issue #8490 

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json [hunk 1]
```diff
@@ -201,7 +201,16 @@
...
     "add-bot": "Add Bot",
-    "search-for-bots": "Search for bots..."
+    "search-for-bots": "Search for bots...",
+    "condition-is-invalid": "Condition is invalid",
+    "rule-name": "Rule Name",
+    "write-your-description": "Write your description",
+    "select-resource": "Select Resources",
+    "name-is-required": "Name is required",
+    "resources-is-required": "Resources are required",
+    "operations-are-required": "Operations are required",
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:211]: For such let's create generic labels like:
```
"field-required": "{{field}} is required",
"field-required-plural": "{{field}} are required"
```

File: openmetadata-ui/src/main/resources/ui/src/pages/PoliciesPage/RuleForm/RuleForm.tsx [hunk 2]
```diff
@@ -108,8 +110,8 @@ const RuleForm: FC<RuleFormProps> = ({ ruleData, setRuleData }) => {
     return option;
   }, [ruleData.resources, policyResources]);
 
-  const getConditionOptions = (funtions: Function[]) => {
-    return funtions.reduce((prev: BaseOptionType[], curr: Function) => {
+  const getConditionOptions = (functions: Function[]) => {
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/PoliciesPage/RuleForm/RuleForm.tsx:113]: what is functions?

Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/pages/PoliciesPage/RuleForm/RuleForm.tsx:113]: functions are condition functions like `isOwner()`.

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json (hunk 1 above)
Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:211]: Sure I will update it.

Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:211]: Then we can use it like this right?
```js
t('label.field-required', { field: 'name' }),
```

File: openmetadata-ui/src/main/resources/ui/src/pages/PoliciesPage/RuleForm/RuleForm.tsx [hunk 3]
```diff
@@ -263,16 +272,17 @@ const RuleForm: FC<RuleFormProps> = ({ ruleData, setRuleData }) => {
         />
       </Form.Item>
       <Form.Item
-        label="Effect:"
+        label={`${t('label.effect')}:`}
         name="ruleEffect"
         rules={[
           {
             required: true,
+            message: t('label.field-required', { field: 'Effect' }),
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/PoliciesPage/RuleForm/RuleForm.tsx:280]:
```suggestion
            message: t('label.field-required', { field: t('label.effect') }),
```

Update same for others as well

Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/pages/PoliciesPage/RuleForm/RuleForm.tsx]: done.

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**